cmd/cli/cmd/install.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"capact.io/capact/internal/cli/capact"
+
+	"capact.io/capact/internal/cli/install"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+// NewInstall returns a cobra.Command for installing Capact in the created env
+func NewInstall() *cobra.Command {
+	opts := install.Options{}
+
+	installCmd := &cobra.Command{
+		Use:   "install [OPTIONS] [SERVER]",
+		Short: "install Capact into target server",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			k8sCfg, err := config.GetConfig()
+			exitOnError(errors.Wrap(err, "while creating k8s config"))
+
+			err = install.Install(cmd.Context(), os.Stdout, k8sCfg, opts)
+			return err
+		},
+	}
+
+	flags := installCmd.Flags()
```

Comment:
I like the consistency between the upgrade and install but I'm thinking how many flags we will need to set up to install Capact from `main`, `release`, `local`, sth what I described here: https://github.com/capactio/capact/issues/328#issuecomment-866799253

Do you have plan for that? Will you add some syntax sugar?

Reply:
I will run some experiments and will post it in the next PR